### DISCUSSION TITLE
Adding Semantics for logs, metrics and traces

### DISCUSSION THREAD

**Discussion by deepkumar-git** [open-telemetry/community#1335] [Community Discussions]
Hi,

We are planning to use OpenTelemetry for our projects. Base data models is fine but if we need to apply additional semantics, should we customize the current semantic convention YAML and use in client side?

**Comment by tigrannajaryan** [comment]
Please clarify what do you mean by "apply additional semantics".

**Reply by deepkumar-git** [reply]
We are working on healthcare domain and has lot of healthcare specific attributes used in metrics space to be harmonized across and limit certain attribute values.. enums etc. (in logs & metrics) so that auto code is limiting developers to use only those values.

**Reply by deepkumar-git** [reply]
Can we customize the semantics and use it in our client side so that back end  also is harmonized  without extending Otel?

**Reply by tigrannajaryan** [reply]
It seems to be a relatively narrow domain compared to most domains OpenTelemetry semantic conventions cover now. You can try to [open an issue](https://github.com/open-telemetry/opentelemetry-specification/issues) to see if there is enough interest from the community to have these conventions in OpenTelemetry.
